include/cell/copy/copy.hpp

```diff
@@ -2,7 +2,7 @@
 
 #include "cuda_utils.hpp"
 
-#include <cute/algorithm/copy.hpp>
```

Comment:
@KuangjuX I've started cleaning up the include relationships. As we continue to refine the copy implementations, this should help resolve the include order issues more effectively. The current separation into static copy, dynamic copy, and copy is somewhat redundant.